game_of_life-java/src/main/java/GameOfLife/Graphics.java

```diff
@@ -24,12 +24,14 @@ public void windowClosing(WindowEvent windowEvent) {
     }
 
     public static void drawCanvas(Canvas canvas, Frame frame) {
+        boolean[][] grid = canvas.getCanvas();
+        java.awt.Graphics graphics = frame.getGraphics();
```

Comment:
Why isn't `java.awt.Graphics` imported?

Reply:
It certainly is in line 3, however, the class is called `Graphics, the FQN of the class must be used to differentiate.